website/i18n/en.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
This file is used to create en.js which is included in siteConfig['en'] for localization strings.

This is also our seed file for language translation.